lib/Beam/Emitter.pm

```diff
@@ -157,7 +157,8 @@ An alias for L</unsubscribe>
 Emit a L<Beam::Event> with the given C<name>. C<event_args> is a list of name => value
 pairs to give to the C<Beam::Event> constructor.
 
-Use the C<class> key in C<event_args> to specify a different Event class.
+Use the C<class> key in C<event_args> to specify a different Event class.  The C<emitter>
```

Comment:
I think we can just leave off this doc update, since it's implied by the previous paragraph. Indeed, we've now simply removed both of the hardwired special-cases to be mere defaults ;)